### DISCUSSION TITLE
BPMN XML export support

### DISCUSSION THREAD

**Discussion by RJanata** [jgraph/drawio#2829] [Ideas]
All our Confluence users do like to use draw.io in our Confluence for various diagrams. However, our analysts and architects use primarily heavier Enterprise Architect. But they'd like to use easy-to-use draw.io for initial drafts and collaboration with other non-EA users, mainly for BPMN diagrams. When the design is done, they'd like to store it in EA.

Therefore, we'd appreciate if draw.io can export its diagrams into BPMN XML format.

Thanks for considering this.

Related issues:
#1238, #1857, #1663

**Comment by FFS-GmbH** [comment]
We'd like to interface with similar tools who speak BPMN XML: 
- [Camunda Modeler](https://camunda.com/download/modeler/) (also available as a Confluence App) 
- [Yaoqiang BPMN Editor ](https://bpmn.sourceforge.net/) 

Both of those are compatible with each other (on a basic level at least, I don't know if this extends to all details) when exchanging "*.bpmn" (XML) files. 

_draw.io status as of April 2025:_ 
- import into draw.io yields no error message, but leaves a blank canvas after import 
- export from draw.io is not provided at all

**Comment by sdykae** [comment]
any updates :(?

[a comment by an automated account is not shown]
